homepage/homepage/app/docs/[framework]/[...slug]/upgrade/0-9-0.mdx

```diff
@@ -0,0 +1,128 @@
+import { ContentByFramework, CodeGroup } from '@/components/forMdx'
+
+export const metadata = { title: "Upgrade to Jazz 0.9.0" };
+
+# Upgrade to Jazz 0.9.0
+
+<h2 className="not-prose text-sm text-stone-600 dark:text-stone-400 mb-5 pb-2 border-b">
+  07 January 2025
+</h2>
+
+<ContentByFramework framework="react">
+  <p>
+    Version 0.9.0 simplifies the application setup and makes Jazz more intellisense friendly by
+    replacing the `createJazzReactApp` API with top-level imports.
+
+    We have also introduced some new API to make testing Jazz components like a breeze. 🌬️
+  </p>
+
+  <h3>New provider setup</h3>
+
+  <p>
+    The `JazzProvider` is now imported from `jazz-react` instead of `createJazzReactApp`.
+
+    To make Typescript happy when you define your own Account schema, you now need to declare it in the `jazz-react` namespace:
+  </p>
+
+  <CodeGroup>
+      {/* prettier-ignore */}
+      ```tsx
+      import { JazzProvider } from "jazz-react";
+      import { MyAppAccount } from "./schema";
+
+      // Remove this:
+      // const Jazz = createJazzReactApp({ AccountSchema: MyAppAccount });
```

Comment:
I think we had some way to cross out lines in our syntax highlighting, check the Guide for an example